### ISSUE TITLE
.dmg file seems to be broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#148]
```
What steps will reproduce the problem?
1. Download .dmg file from http://code.google.com/p/soundflower/
2. Open it
3. Click "Soundflower.pkg" to install

What is the expected output? What do you see instead?
Finder says .pkg file is broken.

What version of the product are you using? On what operating system?
.dmg file has 1.6.4.
The OS is OS X Mountain Lion(OSX 10.8).

Please provide any additional information below.

```

Original issue reported on code.google.com by `quark.be...@gmail.com` on 27 Jul 2012 at 7:46

**Comment by GoogleCodeExporter** [comment]
```
I've made sure the installer can be opened. Thanks for fixing this issue.
```

Original comment by `quark.be...@gmail.com` on 28 Jul 2012 at 5:37

**Comment by GoogleCodeExporter** [comment]
```
should be fixed in 1.6.5
```

Original comment by `t...@electrotap.com` on 31 Jul 2012 at 6:48
- Changed state: **Fixed**
